packages/protocol/analysisManager.ts

```diff
@@ -179,35 +179,6 @@ class AnalysisManager {
     return handler.onFinished?.();
   }
 
-  async runAnalysisBatches<T>(
```

Comment:
:hand: I was actually considering possibly using this in the near future.  Can we leave this in?